rerun-failed-run/README.md

```diff
@@ -0,0 +1,66 @@
+# rerun-failed-run
+
+This reusable workflow can be used to retrigger in GitHub Actions workflows to automatically retrigger them if they failed and contain a specific error message in their build logs.
+
+## Usage
+
+The workflow can be invoked with the aid of the GitHub CLI from another workflow. In case the calling workflow does not belong in the same repository the designated GitHub App Token should be generated in order to allow the invocation. For that purpose, a github app is already set up, and its credentials can be found in the following Vault secret paths:
+
+```
+secret/data/products/infra/ci/retrigger-gha-workflow RETRIGGER_APP_KEY;
+secret/data/products/infra/ci/retrigger-gha-workflow RETRIGGER_APP_ID;
+```
+
+You can customize how many times you want the workflow to retrigger your job by configuring the number of attempts in the job level. See th example provided
+below for more information.
+
+## Inputs
+
+| Input name               | Description                                                                                                                   |
+|--------------------------|-------------------------------------------------------------------------------------------------------------------------------|
+| run_id (required)        | The ID of the workflow run to retrigger.                                                                                      |
+| error_message (required) | Custom error message to look for in the logs                                                                                  |
+| repository (required)    | GitHub repository calling this reusable workflow, in the format ORG/REPO_NAME (example: `camunda/infra-global-github-action`) |
+
+### Workflow Example
+```yaml
+---
+jobs:
+ rerun:
+  runs-on: gcp-core-2-default
+  steps:
+    - name: Echo dummy text
+      run: echo "dummy text"
+
+ rerun-failed-jobs:
+    needs:
+      - job1
+    if: failure() && fromJSON(github.run_attempt) < 3
```

Comment:
let's add an explicit comment here that `fromJSON(github.run_attempt) < 3` is used a limitator to not retry the build more than 3 times (and is very important to have this condition)

Reply:
yeah that is true